conn.go

```diff
@@ -146,3 +209,11 @@ var ddlQueryRegex = regexp.MustCompile(`(?i)^(ALTER|CREATE|DESCRIBE|DROP|MSCK|SH
 func isDDLQuery(query string) bool {
 	return ddlQueryRegex.Match([]byte(query))
 }
+
+func isSelectQuery(query string) bool {
+	return regexp.MustCompile(`(?i)^SELECT`).Match([]byte(query))
```

Comment:
I have a question.
It might be a stupid question though, but I couldn't find the specification of `(?i)` in [the official doc](https://golang.org/pkg/regexp/)...

I guess `(?i)` means "case-insensitive", is it correct?

Reply:
You can use this to ignore uppercase and lowercase.
http://psychedelicnekopunch.com/archives/504